### DISCUSSION TITLE
How to handle Pending transaction

### DISCUSSION THREAD

**Discussion by amisha2012** [ConsenSys/quorum#1658] [Q&A]
HI Team, 
I have a multi-host, two quorum node (each having a validator) private network based on QBFT consensus. Due to a temp memory issue, node 2 was not able to process transactions, and transactions went into a pending state. Blocks were not created in the network. We solved the temp issue and restarted nodes and validators, but still the block is not generated. How can we handle pending state transactions? Kindly suggest.

================
/ $ geth version
Geth
Version: 1.10.2-stable
Git Commit: e5b930e8e9a6ff043b3e71a4e3b852280fe413f0
Quorum Version: 22.7.0
Architecture: amd64
Network Id: 1337
Go Version: go1.16.15
Operating System: linux

================

Thanks!!

**Comment by BrunoAura** [comment]
Hi 

did you find an answer to that question.
I have the same issue

Regards
